browser/mocha/stub.js

```diff
@@ -25,20 +33,17 @@ extendInterfaces('stub', function(context, teardown) {
     var proto = document.createElement(tagName).constructor.prototype;
 
     // For all keys in the implementation to stub with..
-    var keys = Object.keys(implementation);
-    keys.forEach(function(key) {
+    var stubs = Object.keys(implementation).map(function(key) {
       // Stub the method on the element prototype with Sinon:
-      sinon.stub(proto, key, implementation[key]);
+      return sinon.stub(proto, key, implementation[key]);
     });
 
     // After all tests..
     teardown(function() {
       // For all of the keys in the implementation we stubbed..
-      keys.forEach(function(key) {
+      stubs.forEach(function(stub) {
```

Comment:
So from my understanding when sinon stubs a getter method the property isn't given a restore method like with normal function stubs, instead the getter method is the one that has the restore. In order to get to restore I would need to instead get the property descriptor. While this would work I felt that it was needlessly complex since sinon.stub returns the stub with access to the restore method.